get_branch_rank.py

```diff
@@ -0,0 +1,122 @@
+import requests
+from bs4 import BeautifulSoup
```

Comment:
@icyflame I've added a short documentation to use the script and updated the README.md file at https://github.com/Demfier/Get-branch-rank.
If something else is required tell me.